build/teamcity-roachtest-stress.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+set -euxo pipefail
+
+source "$(dirname "${0}")/teamcity-support.sh"
+
+google_credentials="$GOOGLE_EPHEMERAL_CREDENTIALS"
+generate_ssh_key
+log_into_gcloud
+export ROACHPROD_USER=teamcity
+
+build/builder/mkrelease.sh amd64-linux-gnu build bin/workload bin/roachtest bin/roachprod 
+
+build/teamcity-roachtest-invoke.sh \
+  --debug=true \
+  --cloud=gce \
+  --zones=us-central1-b,us-west1-b,europe-west2-b \
+  --count="${COUNT-16}" \
+  --parallelism="${PARALLELISM-16}" \
+  --cpu-quota="${CPUQUOTA-1024}" \
+  --cluster-id="${TC_BUILD_ID}" \
+  --build-tag="${BUILD_TAG}" \
```

Comment:
This does the right thing of using the auto-detection if BUILD_TAG is empty, right?

Reply:
Yep.